runtime.go

```diff
@@ -471,13 +471,33 @@ type SafeURL string
 
 // Script handling.
 
-// SafeScript encodes unknown parameters for safety.
-func SafeScript(functionName string, params ...interface{}) string {
+func safeEncodeScriptParams(escapeHtml bool, params ...interface{}) []string {
 	encodedParams := make([]string, len(params))
 	for i := 0; i < len(encodedParams); i++ {
 		enc, _ := json.Marshal(params[i])
-		encodedParams[i] = EscapeString(string(enc))
+		if escapeHtml {
+			encodedParams[i] = EscapeString(string(enc))
+		} else {
+			encodedParams[i] = string(enc)
+		}
```

Comment:
This could be de-nested.

```suggestion
		if !escapeHtml {
			encodedParams[i] = string(enc)
			continue
		}
		encodedParams[i] = EscapeString(string(enc))
```